src/main/java/com/Service/MyUserDetailService.java

```diff
@@ -15,18 +13,13 @@
     @Autowired
```

Comment:
Can you find a better name for this class? Maybe TrackerSystemUserDetailsService?

Reply:
Agree. I will rename this class.